app-admin/paxtest/paxtest-0.9.15-r2.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="PaX regression test suite"
+HOMEPAGE="https://pax.grsecurity.net"
+SRC_URI="https://grsecurity.net/~spender/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+
+DEPEND="sys-apps/paxctl"
+
+# EI_PAX flags are not strip safe.
+RESTRICT="strip"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-0.9.13-Makefile.patch"
+)
+
+src_prepare() {
+	mv Makefile.psm Makefile
+	default
+	sed -i "s/^CC := gcc/CC := $(tc-getCC)/" Makefile
```

Comment:
We always want a `|| die` on external commands which may fail.

Also, combine the seds with -e.